src/components/RightSidebar/Participants/Participant.vue

```diff
@@ -222,9 +222,10 @@
 				</NcActionButton>
 			</template>
 
+			<NcActionSeparator v-if="showPermissionsOptions && canBeModerated && isPhoneActor" />
```

Comment:
```suggestion
			<NcActionSeparator v-if="canBeModerated && isPhoneActor && showPermissionsOptions" />
```

so it's again "first block && second block"?

Reply:
Both directions are the same and clear ?